.github/workflows/validate.yml

```diff
@@ -36,6 +36,9 @@ jobs:
       - name: Install dependencies
         run: yarn --frozen-lockfile
 
+      - name: Save PR number
+        run: echo ${{ github.event.number }} > pr.txt
```

Comment:
I wonder if this is different than ${{ github.event.pull_request.number }}

Reply:
I think it's the same